src/test/setup.ts

```diff
@@ -27,12 +27,18 @@ const ignoreConsoleErrorMessages = [
    * jsdom doesn't implement scrolling
    */
   'Error: Not implemented: window.scrollTo',
+  /*
+   * complains about not having compenent that throw before render
+   * not being wrapped in <ErrorBoundary>
+  */
+  /(.*)Consider adding an error boundary to your tree to customize error handling behavior.(.*)/,
 ];
 
 const originalConsoleError = console.error;  // tslint:disable-line:no-console
-console.error = (msg: unknown) => {  // tslint:disable-line:no-console
+console.error = (msg: string) => {  // tslint:disable-line:no-console
```

Comment:
your types in here are messed up. its possible to console.error something that isn't a string, and the conditional below does a typecheck on it to see if it is a string. if it makes life easier i'm not opposed to changing the `ignoreConsoleErrorMessages` array to all be regexes